test.html

```diff
@@ -1,59 +1,179 @@
-This is a website
-A sub-title that will encourage action by a user
+<!doctype html>
+<html lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>LDS Youth</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
 
-Let The Youth Lead
-Watch Youth in Kansas City get started
-with on their activities
+        <link rel="apple-touch-icon" href="apple-touch-icon.png">
+        
+        <link rel="stylesheet" href="style/reset.css">
+        <link rel="stylesheet" href="style/style.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+        
+        
+    </head>
 
-Pick a Seat Near Stage
-Enjoy Nature
-Share Photos and Stories
-Find a Temple near you
+<body>
 
-Step 1
-The best step you can take
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dicta nemo unde magnam explicabo eum corporis quod laudantium placeat qui accusantium fugiat voluptate. Expedita voluptates blanditiis sunt quidem dignissimos officiis repellat?
+<div id="container">
+    <header class="max-width">
+        <h1>This is a website</h1>
+        <p>A sub-title that will encourage action by a user</p>
+            <div class="teaser_image">
+                <img src="images/hero1-desktop.jpg" alt="Three young hugging young men">
+            </div>
+            <div class="teaser_content">
+                <p class="teaser_title">Let The Youth Lead</p>
+                <p class="teaser_text">Watch Youth in Kansas City get started with on their activities</p>
+                <a href="#" class="btn">View Short Video</a>
+            </div>
 
-Share Photos and Stories
+        <nav>
+            <ul class="clearfix">
+                <li><a href="#"><img src="icons/fanchart.svg"><span>Pick a Seat Near Stage</span></a></li>
+                <li><a href="#"><img src="icons/trees.svg"><span>Enjoy Nature</span></a></li>
+                <li><a href="#"><img src="icons/camera.svg"><span>Share Photos and Stories</span></a></li>
+                <li><a href="#"><img src="icons/temple.svg"><span>Find a Temple near you</span></a></li>
+            </ul>
+        </nav>
+    </header>
 
-Step 2
-Getting higher on the steps
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dicta nemo unde magnam explicabo eum corporis quod laudantium placeat qui accusantium fugiat voluptate. Expedita voluptates blanditiis sunt quidem dignissimos officiis repellat? Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+    <wrapper>
```

Comment:
Originally I was going to use wrapper to put the tan background color inside of, but I ended up using the article tags instead. I'm pretty sure I didn't end up using wrapper for anything so it could have been removed.